setup.py

```diff
@@ -39,6 +39,7 @@ def read(fname):
                       'ipopt': ['ipopt>=0.1.9'],
                       'dlib': ['dlib>=19.19.0'],
                       'pyswarm': ['pyswarm>=0.6'],
+	                  'cma': ['cma>=3.0.3'],
```

Comment:
sorry, this should be on the same level as the others. The problem is that you have a tab before there. Please use empty spaces " " only (should be the default in pycharm actually ...)

Reply:
For me, it is on the same level...

Reply:
yes, but you have a tab in there, which in your case is represented by 6 (?) spaces, and here by 4, or something like that. therefore, we need to use spaces only.

Reply:
Is it better now? I only used empty spaces and it looks fine to me